packages/docs/src/stories/components/combobox.stories.ts

```diff
@@ -333,23 +333,23 @@ export const Clearable = {
 export const Icons = {
   render: () => html`
     <div class="w-[400px] h-[400px]">
-      <sd-combobox placeholder="Small" size="sm" clearable>
+      <sd-combobox placeholder="Small" size="sm" clearable label="Label">
         <sd-icon slot="left" name="system/image" aria-hidden="true" color="currentColor"></sd-icon>
         ${createColorOptionsHtml()}
         <button slot="right" aria-label="Search">
           <sd-icon library="system" name="magnifying-glass" aria-hidden="true" color="currentColor"></sd-icon>
         </button>
       </sd-combobox>
       <br />
-      <sd-combobox placeholder="Medium" size="md" clearable>
+      <sd-combobox placeholder="Medium" size="md" clearable label="Label">
         <sd-icon slot="left" name="system/image" aria-hidden="true" color="currentColor"></sd-icon>
         ${createColorOptionsHtml()}
         <button slot="right" aria-label="Search">
           <sd-icon library="system" name="magnifying-glass" aria-hidden="true" color="currentColor"></sd-icon>
         </button>
       </sd-combobox>
       <br />
-      <sd-combobox placeholder="Large" size="lg" clearable>
+      <sd-combobox placeholder="Large" size="lg" clearable label="Label">
```

Comment:
Could we just use the label instead of the placeholder? I fear @matuzo complaining, that here everything has the same label and that we misuse the placeholder to provide info. 😆

Reply:
Updated